packages/dokeysto/dokeysto.3.0.2/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+name: "dokeysto"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/dokeysto"
+bug-reports: "https://github.com/UnixJunkie/dokeysto/issues"
+dev-repo: "git+https://github.com/UnixJunkie/dokeysto.git"
+license: "LGPL-2.1"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test"] {with-test}
```

Comment:
```suggestion
  ["dune" "exec" "-p" name "-j" jobs "src/test.exe"] {with-test}
```
`dune build` alone without `-p` or `--profile=release` (as used in `make test`) is not meant for releases and might to break things in the future.

Reply:
Also i would suggest to use the test feature from dune next time: https://dune.readthedocs.io/en/stable/quick-start.html#defining-tests